packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -30,6 +13,8 @@ jest.mock( 'wpcom-proxy-request', () => ( {
 	__esModule: true,
 } ) );
 
+// const TEST_LOCALE = 'test-locale';
```

Comment:
Can you please remove this?

```suggestion
```

Reply:
Good spot! I was actually meaning to use that variable — I uncommented it and used it in https://github.com/Automattic/wp-calypso/pull/50492/commits/565f71dbe20297cf9be7a08a2bb6ac22167eca37